dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -97,18 +103,24 @@ protected boolean isGithubAppEnabledOnRepository(String fullRepoName) {
      */
     protected boolean isGithubAppEnabledOnRepositoryWithGitApi(String fullRepoName) {
         refreshJwtIfNeeded(appId, privateKeyPath);
-        try {
-            gitHub.getApp().getInstallationByRepository(fullRepoName.split("/")[0], fullRepoName.split("/")[1]);
-            return true;
-        } catch (HttpException exception) {
-            if (exception.getResponseCode() != 404) {
-                // Log for any HTTP status code other than 404 Not found. 
-                log.warn("Caught a HTTPException {} while trying to get app installation. Defaulting to False", exception.getMessage());
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            String apiEndpoint = "https://git.soma.salesforce.com/api/v3/repos/" + fullRepoName + "/installation";
+            HttpGet httpGet = new HttpGet(apiEndpoint);
+            httpGet.setHeader("Authorization", jwt);
+            httpGet.setHeader("Accept", "application/vnd.github+json");
+
+            HttpResponse response = httpClient.execute(httpGet);
```

Comment:
I believe http responses need to be closed as well, https://stackoverflow.com/questions/48928930/right-way-to-close-closeablehttpresponse-closeablehttpclient

Reply:
@cjneasbi the reponse is `HttpResponse` type, which is not closable. Only `CloseableHttpResponse` type can. 
```
[ERROR] /Users/hanelliotphan/Desktop/dockerfile-image-update/dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java:[114,21] cannot find symbol
[ERROR]   symbol:   method close()
[ERROR]   location: variable response of type org.apache.http.HttpResponse
```

Reply:
@cjneasbi Made the changes